### PR TITLE
Add Traditional Chinese translation (zh-tr)

### PR DESCRIPTION
**Pull request by PaoMian0806**

### CODE REVIEW

**Comment by RTa-technology** [comment]
I will check today or tomorrow ...
If you don't get a response, please send me a @mention.

File: src/public/locales/zh-tr/top.ftml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+[[li]][# 規則][[ul]]
+[[li]][[[Site Rules|網站守則]]][[/li]]
+[[li]][[[Criticism Policy|評價政策]]][[/li]]
+[[li]][[[Licensing Guide|授權指南]]][[/li]]
+[[li]][[[image-use-policy|圖像使用政策]]][[/li]]
+[[li]][[[deletions-guide|條目刪除守則]]][[/li]]
+[[li]][[[/forum/t-12355929/|頁面刪除申報區]]][[/li]]
+[[/ul]][[/li]]
+[[/ul]][[ul]]
+[[li]][[[contact-staff |聯絡工作人員]]][[ul]]
```
Comment (RTa-technology) [inline src/public/locales/zh-tr/top.ftml:33]:
```
	[[li]][[[contact-staff |聯絡工作人員]]][[/li]][[/ul]]
```

File: src/public/locales/zh-tr/top.ftml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
```
Comment (RTa-technology) [inline src/public/locales/zh-tr/top.ftml:73]:
```
	[[li]][[[contact-staff |聯絡工作人員]]][[/li]][[/ul]]
```

**Comment by RTa-technology** [comment]
please ,rename it from `message.yaml` to `messages.yaml`.

File: src/public/locales/zh-tr/message.yaml
Comment (RTa-technology) [inline src/public/locales/zh-tr/message.yaml:1]: please ,rename it from `message.yaml` to `messages.yaml`.

**Review by RTa-technology — CHANGES_REQUESTED** [review]

**Comment by RTa-technology** [comment]
@PaoMian0806 pls recheck

**Comment by PaoMian0806** [comment]
Done👍

**Review by RTa-technology — APPROVED** [review]

**Pull request merged**